doc/changes/latest.rst

```diff
@@ -60,4 +60,7 @@ Changes
 
 - :bdg-primary:`Doc` The documentation for :func:`~image.threshold_img` has been improved, with more information about which voxels are set to 0 and which ones keep their original values (:gh:`3485` by `Rémi Gau`_).
 
+- :bdg-secondary:`Maint` Extract helper-functions for input-image validation from :func:`~regions.img_to_signals_labels`, :func:`~regions.signals_to_img_labels`, :func:`~regions.img_to_signals_maps` :func:`~regions.signals_to_img_maps` (:gh:`3523` by `Rémi Gau`_ and `Christian Gerloff`_).
```

Comment:
I think this should go in the dev section above. Right now it's under the pre-release changelog

Reply:
good catch

just moved it to the relevant section now.